src/code_reviewer/__init__.py

```
@@ -1,5 +1,3 @@
-"""Enrichment for a pre-defined schema."""
+from code_reviewer.lg_server import graph_non_github
```

Ruff (0.8.2)

Missing docstring in public package

(D104)